src/components/App.js

```diff
@@ -14,6 +15,13 @@ var SUB = new NchanSubscriber(
 export default class App extends React.Component {
   constructor(props) {
     super(props);
+    /**
+     * Get user volume preference from local storage
+     * if not available set to default 0.5.
+     */
+    this.volume = store.get('coderadio-volume')
```

Comment:
could you please change this to this.storageVolume and you could evaluate set this.volume on the next line.